src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -220,44 +220,6 @@ public Task GetPeriodicBackupStatus()
             return Task.CompletedTask;
         }
 
-        [RavenAction("/admin/connection-strings", "GET", AuthorizationStatus.DatabaseAdmin)]
-        public Task GetConnectionStrings()
```

Comment:
It was moved to a different handler but this code left here as well so we had duplicated route "/admin/connection-strings"